src/mars.d

```diff
@@ -101,6 +101,7 @@ Where:
 <option>:
   @<cmdfile>       read arguments from cmdfile
   -allinst         generate code for all template instantiations
+  -bc-ctfe         use the experimental bytecode-engine for ctfe
```

Comment:
> I do not think it harms anything having the switch documented.

Well documented switches have the tendency to stick around, so I agree with David.